### PR TITLE
✨ Create new ingest CLI cmd

### PR DESCRIPTION
**Pull request by znatty22**
```shell
# Generate a new ingest project
$ kidsfirst new --dest_dir ./my_study
```

### CODE REVIEW

**Comment by fiendish** [comment]
> include a sample data file and working extract config?

I don't think it's needed

File: kf_lib_data_ingest/factory/templates/my_study/dataset_ingest_config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    authority: 'dbgap'
+    version: 'v0.p0'
+    release_status: 'Pending'
+    category: 'Cancer'
+    attribution: 'some link'
+
+investigators:
+    -
+        kf_id: 'IG_MEOWMEOW'
+        name: 'John Smith'
```
Comment (fiendish) [inline kf_lib_data_ingest/factory/templates/my_study/dataset_ingest_config.yml]: I think we need to talk more about how investigators get created, what is required, and what combination of values is considered unique.

Comment (znatty22) [reply kf_lib_data_ingest/factory/templates/my_study/dataset_ingest_config.yml]: Yea 👍

File: kf_lib_data_ingest/factory/templates/my_study/extract_configs/extract_config.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"""
+Auto-generated extract config module.
+
+See documentation at
+https://kids-first.github.io/kf-lib-data-ingest/ for information on writing
```
Comment (fiendish) [inline kf_lib_data_ingest/factory/templates/my_study/extract_configs/extract_config.py]: https://kids-first.github.io/kf-lib-data-ingest/tutorial/extract.html

Comment (fiendish) [inline kf_lib_data_ingest/factory/templates/my_study/extract_configs/extract_config.py]:
```suggestion
https://kids-first.github.io/kf-lib-data-ingest/tutorial/extract.html for information on writing
```

File: kf_lib_data_ingest/factory/templates/my_study/dataset_ingest_config.yml (hunk 1 above)
Comment (znatty22) [reply kf_lib_data_ingest/factory/templates/my_study/dataset_ingest_config.yml]: Ingest probably shouldn't be the thing that creates the investigator or study anyway

File: kf_lib_data_ingest/cli.py [hunk 3]
```diff
@@ -68,6 +69,21 @@ def ingest(dataset_ingest_config_path, target_url, auto_transform, use_async):
...
+              'be created. If not specified, the current working dir will '
+              'be used.',
+              type=click.Path(exists=False, file_okay=False, dir_okay=True))
+def create_new_ingest(dest_dir=None):
+    """
+    Create a new ingest package based on the template:
+    kf_lib_data_ingest.factory.templates.study
+    """
+    from kf_lib_data_ingest.factory.generate import new_ingest_pkg
+    new_ingest_pkg(dest_dir)
```
Comment (fiendish) [inline kf_lib_data_ingest/cli.py]: Can this catch the exception and just print the error messge? Full traceback is spammy here

Comment (znatty22) [reply kf_lib_data_ingest/cli.py]: 👍

**Pull request merged**